<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>BSEA control vs IBD in cMD for aerophilicity attributes â€¢ bugphyzzAnalyses</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="BSEA control vs IBD in cMD for aerophilicity attributes">
<meta property="og:description" content="bugphyzzAnalyses">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">bugphyzzAnalyses</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/data_integration_and_stats.html">Data integration and stats</a>
    </li>
    <li>
      <a href="../articles/validation.html">Cross-validation</a>
    </li>
    <li>
      <a href="../articles/subgingival_plaque.html">Known ground truth - Subgingival vs supragingival plaque</a>
    </li>
    <li>
      <a href="../articles/nychanes.html">Reproducibility of previous study - NYCHANES study - bugphyzz</a>
    </li>
    <li>
      <a href="../articles/nychanes2.html">Reproducibility of previous study - NYCHANES study - original annotations</a>
    </li>
    <li>
      <a href="../articles/bsdb_signatures.html">High-throughput ORA with BSDB signatures</a>
    </li>
    <li>
      <a href="../articles/up_vs_down_signatures.html">Odds Ratio - Up vs Down - signatures</a>
    </li>
    <li>
      <a href="../articles/up_vs_down_metasignatures_nointersect.html">Odds Ratio - Up vs Down - metasignatures</a>
    </li>
    <li>
      <a href="../articles/TypicalMicrobiomeSignatures.html">Threhold for TypicalMicrobiomeSignatures</a>
    </li>
    <li>
      <a href="../articles/ibd_enrichment.html">Enrichment in IBD samples</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugphyzzAnalyses/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><link href="ibd_enrichment_files/htmltools-fill-0.5.8/fill.css" rel="stylesheet">
<script src="ibd_enrichment_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><link href="ibd_enrichment_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="ibd_enrichment_files/datatables-binding-0.32/datatables.js"></script><link href="ibd_enrichment_files/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="ibd_enrichment_files/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="ibd_enrichment_files/dt-core-1.13.6/js/jquery.dataTables.min.js"></script><link href="ibd_enrichment_files/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet">
<script src="ibd_enrichment_files/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script><script src="ibd_enrichment_files/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script><script src="ibd_enrichment_files/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script><script src="ibd_enrichment_files/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script><link href="ibd_enrichment_files/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet">
<script src="ibd_enrichment_files/nouislider-7.0.10/jquery.nouislider.min.js"></script><link href="ibd_enrichment_files/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet">
<script src="ibd_enrichment_files/selectize-0.12.0/selectize.min.js"></script><link href="ibd_enrichment_files/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="ibd_enrichment_files/crosstalk-1.2.1/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>BSEA control vs IBD in cMD for aerophilicity
attributes</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/bugphyzzAnalyses/blob/HEAD/vignettes/articles/ibd_enrichment.Rmd" class="external-link"><code>vignettes/articles/ibd_enrichment.Rmd</code></a></small>
      <div class="hidden name"><code>ibd_enrichment.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugphyzzAnalyses" class="external-link">bugphyzzAnalyses</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugphyzz" class="external-link">bugphyzz</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/curatedMetagenomicData" class="external-link">curatedMetagenomicData</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">EnrichmentBrowser</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/rstudio/DT" class="external-link">DT</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://purrr.tidyverse.org/" class="external-link">purrr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tibble.tidyverse.org/" class="external-link">tibble</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="expected-result">Expected result<a class="anchor" aria-label="anchor" href="#expected-result"></a>
</h2>
<ul>
<li>Enrichment of anaerobic bacteria in healthy samples (gut).</li>
<li>Enrichment of facultative anaerobic bacteria in IBD samples
(gut).</li>
</ul>
</div>
<div class="section level2">
<h2 id="data">Data<a class="anchor" aria-label="anchor" href="#data"></a>
</h2>
<p>Use the HMP_2019_bidmdb study in cMD:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dataset_name</span> <span class="op">&lt;-</span> <span class="st">"HallAB_2017.relative_abundance"</span></span>
<span><span class="va">tse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedMetagenomicData/man/curatedMetagenomicData.html" class="external-link">curatedMetagenomicData</a></span><span class="op">(</span></span>
<span>    pattern <span class="op">=</span> <span class="va">dataset_name</span>, </span>
<span>    dryrun <span class="op">=</span> <span class="cn">FALSE</span>, rownames <span class="op">=</span> <span class="st">'NCBI'</span>,</span>
<span>    counts <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span><span class="op">$</span><span class="va">GROUP</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/ifelse.html" class="external-link">ifelse</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span><span class="op">$</span><span class="va">study_condition</span> <span class="op">==</span> <span class="st">'control'</span>, <span class="fl">0</span>, <span class="fl">1</span><span class="op">)</span></span>
<span><span class="co">## Filter taxa with at least 1 count as abundance in at least 20% of samples</span></span>
<span><span class="va">min_n_samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Round.html" class="external-link">round</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">ncol</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span> <span class="op">*</span> <span class="fl">0.2</span><span class="op">)</span></span>
<span><span class="va">tse_subset</span> <span class="op">&lt;-</span> <span class="va">tse</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowSums</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span> <span class="op">&gt;=</span> <span class="fl">1</span><span class="op">)</span> <span class="op">&gt;=</span> <span class="va">min_n_samples</span>,<span class="op">]</span></span>
<span><span class="va">tse_subset</span></span>
<span><span class="co">#&gt; class: TreeSummarizedExperiment </span></span>
<span><span class="co">#&gt; dim: 117 259 </span></span>
<span><span class="co">#&gt; metadata(1): agglomerated_by_rank</span></span>
<span><span class="co">#&gt; assays(1): relative_abundance</span></span>
<span><span class="co">#&gt; rownames(117): 853 820 ... 1262992 329854</span></span>
<span><span class="co">#&gt; rowData names(7): superkingdom phylum ... genus species</span></span>
<span><span class="co">#&gt; colnames(259): p8582_mo1 p8582_mo10 ... SKST041_2_G103027</span></span>
<span><span class="co">#&gt;   SKST041_3_G103028</span></span>
<span><span class="co">#&gt; colData names(25): study_name subject_id ... SCCAI GROUP</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span>
<span><span class="co">#&gt; rowLinks: a LinkDataFrame (117 rows)</span></span>
<span><span class="co">#&gt; rowTree: 1 phylo tree(s) (10430 leaves)</span></span>
<span><span class="co">#&gt; colLinks: NULL</span></span>
<span><span class="co">#&gt; colTree: NULL</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="bugphyzz-data">bugphyzz data<a class="anchor" aria-label="anchor" href="#bugphyzz-data"></a>
</h2>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">bp</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/bugphyzz/man/importBugphyzz.html" class="external-link">importBugphyzz</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">aer</span> <span class="op">&lt;-</span> <span class="va">bp</span><span class="op">$</span><span class="va">aerophilicity</span></span>
<span><span class="va">sp_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/bugphyzz/man/makeSignatures.html" class="external-link">makeSignatures</a></span><span class="op">(</span>dat <span class="op">=</span> <span class="va">aer</span>, tax_id_type <span class="op">=</span> <span class="st">'NCBI_ID'</span>, tax_level <span class="op">=</span> <span class="st">'species'</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://purrr.tidyverse.org/reference/keep.html" class="external-link">discard</a></span><span class="op">(</span><span class="va">is.null</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="differential-abundance">Differential abundance<a class="anchor" aria-label="anchor" href="#differential-abundance"></a>
</h2>
<p>DA:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse_subset</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse_subset</span><span class="op">)</span> <span class="op">+</span> <span class="fl">1</span></span>
<span><span class="va">edger</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/deAna.html" class="external-link">deAna</a></span><span class="op">(</span></span>
<span>    expr <span class="op">=</span> <span class="va">tse_subset</span>, de.method <span class="op">=</span> <span class="st">'edgeR'</span>, padj.method <span class="op">=</span> <span class="st">'BH'</span>,</span>
<span>    filter.by.expr <span class="op">=</span> <span class="cn">FALSE</span> </span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">edger</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 117 259</span></span></code></pre></div>
<blockquote>
<p>Almost all bacteria found in the SummarizedExperiment are anaerobic.
Maybe a different approach would be more convenient, like merging
several datasets in CuratedMetagenomicData.</p>
</blockquote>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">lapply</a></span><span class="op">(</span><span class="va">sp_sigs</span>, <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/base/sum.html" class="external-link">sum</a></span><span class="op">(</span><span class="va">x</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">rownames</a></span><span class="op">(</span><span class="va">edger</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; $`bugphyzz:aerophilicity|aerobic`</span></span>
<span><span class="co">#&gt; [1] 4</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`bugphyzz:aerophilicity|anaerobic`</span></span>
<span><span class="co">#&gt; [1] 88</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`bugphyzz:aerophilicity|facultatively anaerobic`</span></span>
<span><span class="co">#&gt; [1] 5</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="enrichment-gsea---bugphyzz">Enrichment (GSEA) - bugphyzz:<a class="anchor" aria-label="anchor" href="#enrichment-gsea---bugphyzz"></a>
</h2>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">edger</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/limmaVoom.html">limmaVoom</a></span><span class="op">(</span><span class="va">edger</span><span class="op">)</span></span>
<span><span class="va">gsea</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'gsea'</span>, se <span class="op">=</span> <span class="va">edger</span>, gs <span class="op">=</span> <span class="va">sp_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">1000</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span></span>
<span><span class="op">)</span></span>
<span><span class="va">gsea_res</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="va">gsea</span><span class="op">$</span><span class="va">res.tbl</span><span class="op">)</span></span>
<span></span>
<span><span class="va">caption1</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>  <span class="st">'Table 1. GSEA results comparing the gut microbiome of subjects with</span></span>
<span><span class="st">  a normal gut micriobiome (healthy controls) and subjects with IBD.</span></span>
<span><span class="st">  ES = Enrichment score, NES = Normalized Enrichment Score. Positive and</span></span>
<span><span class="st">  negative scores (ES and NES) indicate enrichment in the IBD and</span></span>
<span><span class="st">  healthy control samples, respectively. Bug sets marked with an asterik (*) </span></span>
<span><span class="st">  indicate significant enrichment results (adjusted p-value &lt; 0.1). </span></span>
<span><span class="st">  Benjamini &amp; Hochberg (BH) was used for P value adjustment.'</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">gsea_res</span>,</span>
<span>  filter <span class="op">=</span> <span class="st">'top'</span>,</span>
<span>  rownames <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  extensions <span class="op">=</span> <span class="st">'Buttons'</span>,</span>
<span>  options <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    dom <span class="op">=</span> <span class="st">'Bft'</span>,</span>
<span>    buttons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="st">'copy'</span>, <span class="st">'print'</span><span class="op">)</span>,</span>
<span>    iDisplayLength <span class="op">=</span> <span class="fl">10</span>,</span>
<span>    keys <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    autoWidth <span class="op">=</span> <span class="cn">TRUE</span></span>
<span>  <span class="op">)</span>,</span>
<span>  caption <span class="op">=</span> <span class="fu">htmltools</span><span class="fu">::</span><span class="va"><a href="https://rstudio.github.io/htmltools/reference/builder.html" class="external-link">tags</a></span><span class="op">$</span><span class="fu">caption</span><span class="op">(</span></span>
<span>    style <span class="op">=</span> <span class="st">'caption-side: top; text-align: left;'</span>,</span>
<span>    <span class="va">caption1</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item" id="htmlwidget-2a850a22d5b2462550d2" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2a850a22d5b2462550d2">{"x":{"filter":"top","vertical":false,"filterHTML":"<tr>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"-0.541\" data-max=\"0.716\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"-1.2\" data-max=\"1.31\" data-scale=\"2\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0.187\" data-max=\"0.227\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\" disabled=\"\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0\" data-max=\"1\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n<\/tr>","extensions":["Buttons"],"caption":"<caption style=\"caption-side: top; text-align: left;\">Table 1. GSEA results comparing the gut microbiome of subjects with\n  a normal gut micriobiome (healthy controls) and subjects with IBD.\n  ES = Enrichment score, NES = Normalized Enrichment Score. Positive and\n  negative scores (ES and NES) indicate enrichment in the IBD and\n  healthy control samples, respectively. Bug sets marked with an asterik (*) \n  indicate significant enrichment results (adjusted p-value &lt; 0.1). \n  Benjamini &amp; Hochberg (BH) was used for P value adjustment.<\/caption>","data":[["bugphyzz:aerophilicity|facultatively anaerobic","bugphyzz:aerophilicity|anaerobic"],[0.716,-0.541],[1.31,-1.2],[0.187,0.227],[0.227,0.227]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>GENE.SET<\/th>\n      <th>ES<\/th>\n      <th>NES<\/th>\n      <th>PVAL<\/th>\n      <th>ADJ.PVAL<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bft","buttons":["copy","print"],"iDisplayLength":10,"keys":true,"autoWidth":true,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"name":"GENE.SET","targets":0},{"name":"ES","targets":1},{"name":"NES","targets":2},{"name":"PVAL","targets":3},{"name":"ADJ.PVAL","targets":4}],"order":[],"orderClasses":false,"orderCellsTop":true},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="import-biosis-data">Import biosis data<a class="anchor" aria-label="anchor" href="#import-biosis-data"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">biosis</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importBiosis.html">importBiosis</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">row_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">tse_subset</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">row_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://tibble.tidyverse.org/reference/rownames.html" class="external-link">rownames_to_column</a></span><span class="op">(</span><span class="va">row_data</span>, var <span class="op">=</span> <span class="st">'rowname'</span><span class="op">)</span></span>
<span><span class="co"># row_data &lt;- modify_at(row_data, 'genus', ~ as.character(.x))</span></span>
<span><span class="va">row_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate-joins.html" class="external-link">left_join</a></span><span class="op">(</span><span class="va">row_data</span>, <span class="va">biosis</span>, by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'genus'</span> <span class="op">=</span> <span class="st">'GenusID'</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">biosis_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/split.html" class="external-link">split</a></span><span class="op">(</span><span class="va">row_data</span>, <span class="fu"><a href="https://rdrr.io/r/base/factor.html" class="external-link">factor</a></span><span class="op">(</span><span class="va">row_data</span><span class="op">$</span><span class="va">Attribute</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">biosis_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://purrr.tidyverse.org/reference/map.html" class="external-link">map</a></span><span class="op">(</span><span class="va">biosis_sigs</span>, <span class="op">~</span> <span class="fu"><a href="https://rdrr.io/r/base/unique.html" class="external-link">unique</a></span><span class="op">(</span><span class="va">.x</span><span class="op">$</span><span class="va">rowname</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">biosis_sigs</span></span>
<span><span class="co">#&gt; $anaerobic</span></span>
<span><span class="co">#&gt;  [1] "820"     "301301"  "357276"  "39491"   "40520"   "418240"  "28116"  </span></span>
<span><span class="co">#&gt;  [8] "674529"  "360807"  "371601"  "47678"   "74426"   "818"     "1262986"</span></span>
<span><span class="co">#&gt; [15] "821"     "1262756" "1262989" "301302"  "166486"  "39492"   "1263005"</span></span>
<span><span class="co">#&gt; [22] "1535"    "165179"  "147207"  "46506"   "33035"   "53443"   "29466"  </span></span>
<span><span class="co">#&gt; [29] "39778"   "1911679" "39777"   "218538"  "1532"    "1262948" "817"    </span></span>
<span><span class="co">#&gt; [36] "338188"  "1262992" "329854" </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $facultatively_anaerobic</span></span>
<span><span class="co">#&gt;  [1] "39491"   "1304"    "1262986" "729"     "1318"    "1262989" "39492"  </span></span>
<span><span class="co">#&gt;  [8] "28026"   "1263005" "1535"    "1308"    "216816"  "1680"    "1681"   </span></span>
<span><span class="co">#&gt; [15] "1358"    "1262992"</span></span></code></pre></div>
<blockquote>
<p>No aerobic taxa, which I think should be expected?</p>
</blockquote>
</div>
<div class="section level2">
<h2 id="enrichemnt-gsea---biosis">Enrichemnt (GSEA) - biosis<a class="anchor" aria-label="anchor" href="#enrichemnt-gsea---biosis"></a>
</h2>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">gsea2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'gsea'</span>, se <span class="op">=</span> <span class="va">edger</span>, gs <span class="op">=</span> <span class="va">biosis_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">1000</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span></span>
<span><span class="op">)</span></span>
<span><span class="va">gsea2_res</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="va">gsea2</span><span class="op">$</span><span class="va">res.tbl</span><span class="op">)</span></span>
<span></span>
<span><span class="va">caption2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>  <span class="st">'Table 2. GSEA results comparing the gut microbiome of subjects with</span></span>
<span><span class="st">  a normal gut micriobiome (healthy controls) and subjects with IBD using</span></span>
<span><span class="st">  biosis (nychanes) signatures.</span></span>
<span><span class="st">  ES = Enrichment score, NES = Normalized Enrichment Score. Positive and</span></span>
<span><span class="st">  negative scores (ES and NES) indicate enrichment in the IBD and</span></span>
<span><span class="st">  healthy control samples, respectively. Bug sets marked with an asterik (*) </span></span>
<span><span class="st">  indicate significant enrichment results (adjusted p-value &lt; 0.1). </span></span>
<span><span class="st">  Benjamini &amp; Hochberg (BH) was used for P value adjustment.'</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">gsea2_res</span>,</span>
<span>  filter <span class="op">=</span> <span class="st">'top'</span>,</span>
<span>  rownames <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  extensions <span class="op">=</span> <span class="st">'Buttons'</span>,</span>
<span>  options <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    dom <span class="op">=</span> <span class="st">'Bft'</span>,</span>
<span>    buttons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="st">'copy'</span>, <span class="st">'print'</span><span class="op">)</span>,</span>
<span>    iDisplayLength <span class="op">=</span> <span class="fl">10</span>,</span>
<span>    keys <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    autoWidth <span class="op">=</span> <span class="cn">TRUE</span></span>
<span>  <span class="op">)</span>,</span>
<span>  caption <span class="op">=</span> <span class="fu">htmltools</span><span class="fu">::</span><span class="va"><a href="https://rstudio.github.io/htmltools/reference/builder.html" class="external-link">tags</a></span><span class="op">$</span><span class="fu">caption</span><span class="op">(</span></span>
<span>    style <span class="op">=</span> <span class="st">'caption-side: top; text-align: left;'</span>,</span>
<span>    <span class="va">caption2</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item" id="htmlwidget-564e3bcfb7b8a52914d6" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-564e3bcfb7b8a52914d6">{"x":{"filter":"top","vertical":false,"filterHTML":"<tr>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"-0.472\" data-max=\"0.309\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"-1.24\" data-max=\"0.826\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0.188\" data-max=\"0.67\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none;position: absolute;width: 200px;opacity: 1\">\n      <div data-min=\"0.376\" data-max=\"0.67\" data-scale=\"3\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n<\/tr>","extensions":["Buttons"],"caption":"<caption style=\"caption-side: top; text-align: left;\">Table 2. GSEA results comparing the gut microbiome of subjects with\n  a normal gut micriobiome (healthy controls) and subjects with IBD using\n  biosis (nychanes) signatures.\n  ES = Enrichment score, NES = Normalized Enrichment Score. Positive and\n  negative scores (ES and NES) indicate enrichment in the IBD and\n  healthy control samples, respectively. Bug sets marked with an asterik (*) \n  indicate significant enrichment results (adjusted p-value &lt; 0.1). \n  Benjamini &amp; Hochberg (BH) was used for P value adjustment.<\/caption>","data":[["anaerobic","facultatively_anaerobic"],[-0.472,0.309],[-1.24,0.826],[0.188,0.67],[0.376,0.67]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>GENE.SET<\/th>\n      <th>ES<\/th>\n      <th>NES<\/th>\n      <th>PVAL<\/th>\n      <th>ADJ.PVAL<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Bft","buttons":["copy","print"],"iDisplayLength":10,"keys":true,"autoWidth":true,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"name":"GENE.SET","targets":0},{"name":"ES","targets":1},{"name":"NES","targets":2},{"name":"PVAL","targets":3},{"name":"ADJ.PVAL","targets":4}],"order":[],"orderClasses":false,"orderCellsTop":true},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="session-information">Session Information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.3.2 (2023-10-31)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 22.04.3 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; time zone: Etc/UTC</span></span>
<span><span class="co">#&gt; tzcode source: system (glibc)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] tibble_3.2.1                    purrr_1.0.2                    </span></span>
<span><span class="co">#&gt;  [3] DT_0.32                         EnrichmentBrowser_2.32.0       </span></span>
<span><span class="co">#&gt;  [5] graph_1.80.0                    dplyr_1.1.4                    </span></span>
<span><span class="co">#&gt;  [7] curatedMetagenomicData_3.10.0   TreeSummarizedExperiment_2.10.0</span></span>
<span><span class="co">#&gt;  [9] Biostrings_2.70.3               XVector_0.42.0                 </span></span>
<span><span class="co">#&gt; [11] SingleCellExperiment_1.24.0     SummarizedExperiment_1.32.0    </span></span>
<span><span class="co">#&gt; [13] Biobase_2.62.0                  GenomicRanges_1.54.1           </span></span>
<span><span class="co">#&gt; [15] GenomeInfoDb_1.38.8             IRanges_2.36.0                 </span></span>
<span><span class="co">#&gt; [17] S4Vectors_0.40.2                BiocGenerics_0.48.1            </span></span>
<span><span class="co">#&gt; [19] MatrixGenerics_1.14.0           matrixStats_1.2.0              </span></span>
<span><span class="co">#&gt; [21] bugphyzz_0.99.0                 bugphyzzAnalyses_0.1.0         </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;   [1] jsonlite_1.8.8                MultiAssayExperiment_1.28.0  </span></span>
<span><span class="co">#&gt;   [3] magrittr_2.0.3                ggbeeswarm_0.7.2             </span></span>
<span><span class="co">#&gt;   [5] rmarkdown_2.26                fs_1.6.3                     </span></span>
<span><span class="co">#&gt;   [7] zlibbioc_1.48.2               ragg_1.3.0                   </span></span>
<span><span class="co">#&gt;   [9] vctrs_0.6.5                   memoise_2.0.1                </span></span>
<span><span class="co">#&gt;  [11] DelayedMatrixStats_1.24.0     RCurl_1.98-1.14              </span></span>
<span><span class="co">#&gt;  [13] htmltools_0.5.8               S4Arrays_1.2.1               </span></span>
<span><span class="co">#&gt;  [15] AnnotationHub_3.10.0          curl_5.2.1                   </span></span>
<span><span class="co">#&gt;  [17] BiocNeighbors_1.20.2          SparseArray_1.2.4            </span></span>
<span><span class="co">#&gt;  [19] sass_0.4.9                    bslib_0.6.2                  </span></span>
<span><span class="co">#&gt;  [21] htmlwidgets_1.6.4             desc_1.4.3                   </span></span>
<span><span class="co">#&gt;  [23] plyr_1.8.9                    DECIPHER_2.30.0              </span></span>
<span><span class="co">#&gt;  [25] cachem_1.0.8                  igraph_2.0.3                 </span></span>
<span><span class="co">#&gt;  [27] mime_0.12                     lifecycle_1.0.4              </span></span>
<span><span class="co">#&gt;  [29] pkgconfig_2.0.3               rsvd_1.0.5                   </span></span>
<span><span class="co">#&gt;  [31] Matrix_1.6-1.1                R6_2.5.1                     </span></span>
<span><span class="co">#&gt;  [33] fastmap_1.1.1                 GenomeInfoDbData_1.2.11      </span></span>
<span><span class="co">#&gt;  [35] shiny_1.8.0                   digest_0.6.35                </span></span>
<span><span class="co">#&gt;  [37] colorspace_2.1-0              AnnotationDbi_1.64.1         </span></span>
<span><span class="co">#&gt;  [39] scater_1.30.1                 irlba_2.3.5.1                </span></span>
<span><span class="co">#&gt;  [41] ExperimentHub_2.10.0          crosstalk_1.2.1              </span></span>
<span><span class="co">#&gt;  [43] textshaping_0.3.7             RSQLite_2.3.5                </span></span>
<span><span class="co">#&gt;  [45] vegan_2.6-4                   beachmat_2.18.1              </span></span>
<span><span class="co">#&gt;  [47] filelock_1.0.3                fansi_1.0.6                  </span></span>
<span><span class="co">#&gt;  [49] mgcv_1.9-0                    httr_1.4.7                   </span></span>
<span><span class="co">#&gt;  [51] abind_1.4-5                   compiler_4.3.2               </span></span>
<span><span class="co">#&gt;  [53] withr_3.0.0                   bit64_4.0.5                  </span></span>
<span><span class="co">#&gt;  [55] BiocParallel_1.36.0           viridis_0.6.5                </span></span>
<span><span class="co">#&gt;  [57] DBI_1.2.2                     MASS_7.3-60                  </span></span>
<span><span class="co">#&gt;  [59] rappdirs_0.3.3                DelayedArray_0.28.0          </span></span>
<span><span class="co">#&gt;  [61] bluster_1.12.0                permute_0.9-7                </span></span>
<span><span class="co">#&gt;  [63] tools_4.3.2                   vipor_0.4.7                  </span></span>
<span><span class="co">#&gt;  [65] beeswarm_0.4.0                ape_5.7-1                    </span></span>
<span><span class="co">#&gt;  [67] interactiveDisplayBase_1.40.0 httpuv_1.6.14                </span></span>
<span><span class="co">#&gt;  [69] glue_1.7.0                    nlme_3.1-163                 </span></span>
<span><span class="co">#&gt;  [71] promises_1.2.1                grid_4.3.2                   </span></span>
<span><span class="co">#&gt;  [73] mia_1.10.0                    reshape2_1.4.4               </span></span>
<span><span class="co">#&gt;  [75] cluster_2.1.4                 generics_0.1.3               </span></span>
<span><span class="co">#&gt;  [77] gtable_0.3.4                  tidyr_1.3.1                  </span></span>
<span><span class="co">#&gt;  [79] BiocSingular_1.18.0           ScaledMatrix_1.10.0          </span></span>
<span><span class="co">#&gt;  [81] utf8_1.2.4                    stringr_1.5.1                </span></span>
<span><span class="co">#&gt;  [83] ggrepel_0.9.5                 BiocVersion_3.18.1           </span></span>
<span><span class="co">#&gt;  [85] pillar_1.9.0                  limma_3.58.1                 </span></span>
<span><span class="co">#&gt;  [87] yulab.utils_0.1.4             later_1.3.2                  </span></span>
<span><span class="co">#&gt;  [89] splines_4.3.2                 BiocFileCache_2.10.1         </span></span>
<span><span class="co">#&gt;  [91] treeio_1.26.0                 lattice_0.22-5               </span></span>
<span><span class="co">#&gt;  [93] bit_4.0.5                     annotate_1.80.0              </span></span>
<span><span class="co">#&gt;  [95] tidyselect_1.2.1              DirichletMultinomial_1.44.0  </span></span>
<span><span class="co">#&gt;  [97] locfit_1.5-9.9                scuttle_1.12.0               </span></span>
<span><span class="co">#&gt;  [99] knitr_1.45                    gridExtra_2.3                </span></span>
<span><span class="co">#&gt; [101] edgeR_4.0.16                  xfun_0.43                    </span></span>
<span><span class="co">#&gt; [103] statmod_1.5.0                 KEGGgraph_1.62.0             </span></span>
<span><span class="co">#&gt; [105] stringi_1.8.3                 lazyeval_0.2.2               </span></span>
<span><span class="co">#&gt; [107] yaml_2.3.8                    evaluate_0.23                </span></span>
<span><span class="co">#&gt; [109] codetools_0.2-19              Rgraphviz_2.46.0             </span></span>
<span><span class="co">#&gt; [111] BiocManager_1.30.22           cli_3.6.2                    </span></span>
<span><span class="co">#&gt; [113] xtable_1.8-4                  systemfonts_1.0.6            </span></span>
<span><span class="co">#&gt; [115] munsell_0.5.0                 jquerylib_0.1.4              </span></span>
<span><span class="co">#&gt; [117] Rcpp_1.0.12                   dbplyr_2.5.0                 </span></span>
<span><span class="co">#&gt; [119] png_0.1-8                     XML_3.99-0.16.1              </span></span>
<span><span class="co">#&gt; [121] parallel_4.3.2                ellipsis_0.3.2               </span></span>
<span><span class="co">#&gt; [123] pkgdown_2.0.7                 ggplot2_3.5.0                </span></span>
<span><span class="co">#&gt; [125] blob_1.2.4                    sparseMatrixStats_1.14.0     </span></span>
<span><span class="co">#&gt; [127] bitops_1.0-7                  GSEABase_1.64.0              </span></span>
<span><span class="co">#&gt; [129] decontam_1.22.0               viridisLite_0.4.2            </span></span>
<span><span class="co">#&gt; [131] tidytree_0.4.6                scales_1.3.0                 </span></span>
<span><span class="co">#&gt; [133] crayon_1.5.2                  rlang_1.1.3                  </span></span>
<span><span class="co">#&gt; [135] KEGGREST_1.42.0</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Samuel Gamboa.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
