<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MSEA control vs IBD in cMD for aerophilicity attributes â€¢ bugphyzzAnalyses</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MSEA control vs IBD in cMD for aerophilicity attributes">
<meta property="og:description" content="bugphyzzAnalyses">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">bugphyzzAnalyses</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/data_integration_and_stats.html">Data integration and stats</a>
    </li>
    <li>
      <a href="../articles/subgingival_plaque.html">Known ground truth - Subgingival vs supragingival plaque</a>
    </li>
    <li>
      <a href="../articles/nychanes.html">Reproducibility of previous study - NYCHANES study</a>
    </li>
    <li>
      <a href="../articles/high_throughput_msea_bsdb.html">High-throughput MSAE with BugSigDB sets</a>
    </li>
    <li>
      <a href="../articles/ibd_enrichment.html">Enrichment of IBD samples</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/bugphyzzAnalyses/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MSEA control vs IBD in cMD for aerophilicity
attributes</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/bugphyzzAnalyses/blob/HEAD/vignettes/articles/ibd_enrichment.Rmd" class="external-link"><code>vignettes/articles/ibd_enrichment.Rmd</code></a></small>
      <div class="hidden name"><code>ibd_enrichment.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/bugphyzz" class="external-link">bugphyzz</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/curatedMetagenomicData" class="external-link">curatedMetagenomicData</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">EnrichmentBrowser</span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="expected-result">Expected result<a class="anchor" aria-label="anchor" href="#expected-result"></a>
</h2>
<ul>
<li>Enrichment of anaerobic bacteria in healthy samples (gut).</li>
<li>Enrichment of facultative anaerobic bacteria in IBD samples
(gut).</li>
</ul>
</div>
<div class="section level2">
<h2 id="data">Data<a class="anchor" aria-label="anchor" href="#data"></a>
</h2>
<p>Use the HMP_2019_bidmdb study in cMD:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dataset_name</span> <span class="op">&lt;-</span> <span class="st">"HallAB_2017.relative_abundance"</span></span>
<span><span class="va">tse</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/curatedMetagenomicData/man/curatedMetagenomicData.html" class="external-link">curatedMetagenomicData</a></span><span class="op">(</span></span>
<span>    pattern <span class="op">=</span> <span class="va">dataset_name</span>, </span>
<span>    dryrun <span class="op">=</span> <span class="cn">FALSE</span>, rownames <span class="op">=</span> <span class="st">'NCBI'</span>,</span>
<span>    counts <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span><span class="op">$</span><span class="va">GROUP</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/ifelse.html" class="external-link">ifelse</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span><span class="op">$</span><span class="va">study_condition</span> <span class="op">==</span> <span class="st">'control'</span>, <span class="fl">0</span>, <span class="fl">1</span><span class="op">)</span></span>
<span><span class="va">tse</span> </span>
<span><span class="co">#&gt; class: TreeSummarizedExperiment </span></span>
<span><span class="co">#&gt; dim: 503 259 </span></span>
<span><span class="co">#&gt; metadata(1): agglomerated_by_rank</span></span>
<span><span class="co">#&gt; assays(1): relative_abundance</span></span>
<span><span class="co">#&gt; rownames(503): 853 820 ... 172901 1262744</span></span>
<span><span class="co">#&gt; rowData names(7): superkingdom phylum ... genus species</span></span>
<span><span class="co">#&gt; colnames(259): p8582_mo1 p8582_mo10 ... SKST041_2_G103027</span></span>
<span><span class="co">#&gt;   SKST041_3_G103028</span></span>
<span><span class="co">#&gt; colData names(25): study_name subject_id ... SCCAI GROUP</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span>
<span><span class="co">#&gt; rowLinks: a LinkDataFrame (503 rows)</span></span>
<span><span class="co">#&gt; rowTree: 1 phylo tree(s) (10430 leaves)</span></span>
<span><span class="co">#&gt; colLinks: NULL</span></span>
<span><span class="co">#&gt; colTree: NULL</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="bugphyzz-data">bugphyzz data<a class="anchor" aria-label="anchor" href="#bugphyzz-data"></a>
</h2>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">bp</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/bugphyzz/man/importBugphyzz.html" class="external-link">importBugphyzz</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">bp</span> <span class="op">&lt;-</span> <span class="va">bp</span> <span class="op">|&gt;</span> </span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">Attribute_group</span> <span class="op">==</span> <span class="st">'aerophilicity'</span><span class="op">)</span></span>
<span><span class="va">sp_sigs</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/bugphyzz/man/getBugphyzzSignatures.html" class="external-link">getBugphyzzSignatures</a></span><span class="op">(</span></span>
<span>  df <span class="op">=</span> <span class="va">bp</span>, tax.id.type <span class="op">=</span> <span class="st">'NCBI_ID'</span>, tax.level <span class="op">=</span> <span class="st">'species'</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="deseq2-ora-and-gsea">DESEQ2 + ORA and GSEA<a class="anchor" aria-label="anchor" href="#deseq2-ora-and-gsea"></a>
</h2>
<p>DA:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse</span>, <span class="st">'relab_plus_1'</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse</span>, <span class="st">'relative_abundance'</span><span class="op">)</span> <span class="op">+</span> <span class="fl">1</span></span>
<span><span class="va">deseq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/deAna.html" class="external-link">deAna</a></span><span class="op">(</span></span>
<span>    expr <span class="op">=</span> <span class="va">tse</span>, de.method <span class="op">=</span> <span class="st">'DESeq2'</span>, padj.method <span class="op">=</span> <span class="st">'BH'</span>, </span>
<span>    assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">deseq</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 126 259</span></span></code></pre></div>
<p>ORA:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">deseq_ora</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'ora'</span>, se <span class="op">=</span> <span class="va">deseq</span>, gs <span class="op">=</span> <span class="va">sp_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span>, assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/eaBrowse.html" class="external-link">gsRanking</a></span><span class="op">(</span>res <span class="op">=</span> <span class="va">deseq_ora</span>, signif.only <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="co">#&gt; DataFrame with 4 rows and 5 columns</span></span>
<span><span class="co">#&gt;                 GENE.SET  NR.GENES NR.SIG.GENES      PVAL  ADJ.PVAL</span></span>
<span><span class="co">#&gt;              &lt;character&gt; &lt;numeric&gt;    &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;</span></span>
<span><span class="co">#&gt; 1 aerophilicity:facult..        29           20     0.611         1</span></span>
<span><span class="co">#&gt; 2  aerophilicity:aerobic        27           18     0.718         1</span></span>
<span><span class="co">#&gt; 3 aerophilicity:aeroto..        23           14     0.888         1</span></span>
<span><span class="co">#&gt; 4 aerophilicity:anaero..       111           75     1.000         1</span></span></code></pre></div>
<p>GSEA:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">deseq_gsea</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'gsea'</span>, se <span class="op">=</span> <span class="va">deseq</span>, gs <span class="op">=</span> <span class="va">sp_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span>, assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/eaBrowse.html" class="external-link">gsRanking</a></span><span class="op">(</span>res <span class="op">=</span> <span class="va">deseq_gsea</span>, signif.only <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="co">#&gt; DataFrame with 4 rows and 3 columns</span></span>
<span><span class="co">#&gt;                 GENE.SET      PVAL  ADJ.PVAL</span></span>
<span><span class="co">#&gt;              &lt;character&gt; &lt;numeric&gt; &lt;numeric&gt;</span></span>
<span><span class="co">#&gt; 1  aerophilicity:aerobic  0.000360  0.000762</span></span>
<span><span class="co">#&gt; 2 aerophilicity:facult..  0.000381  0.000762</span></span>
<span><span class="co">#&gt; 3 aerophilicity:anaero..  0.001980  0.002640</span></span>
<span><span class="co">#&gt; 4 aerophilicity:aeroto..  0.004690  0.004690</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="edger">edgeR<a class="anchor" aria-label="anchor" href="#edger"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">edger</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/deAna.html" class="external-link">deAna</a></span><span class="op">(</span></span>
<span>    expr <span class="op">=</span> <span class="va">tse</span>, de.method <span class="op">=</span> <span class="st">'edgeR'</span>, padj.method <span class="op">=</span> <span class="st">'BH'</span>, </span>
<span>    assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>ORA:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">edger_ora</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'ora'</span>, se <span class="op">=</span> <span class="va">edger</span>, gs <span class="op">=</span> <span class="va">sp_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span>, assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/eaBrowse.html" class="external-link">gsRanking</a></span><span class="op">(</span><span class="va">edger_ora</span>, signif.only <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="co">#&gt; DataFrame with 4 rows and 5 columns</span></span>
<span><span class="co">#&gt;                 GENE.SET  NR.GENES NR.SIG.GENES      PVAL  ADJ.PVAL</span></span>
<span><span class="co">#&gt;              &lt;character&gt; &lt;numeric&gt;    &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;</span></span>
<span><span class="co">#&gt; 1 aerophilicity:anaero..       111           72     0.924     0.987</span></span>
<span><span class="co">#&gt; 2  aerophilicity:aerobic        27           15     0.933     0.987</span></span>
<span><span class="co">#&gt; 3 aerophilicity:facult..        29           16     0.946     0.987</span></span>
<span><span class="co">#&gt; 4 aerophilicity:aeroto..        23           11     0.987     0.987</span></span></code></pre></div>
<p>GSEA:</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">edger_gsea</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/sbea.html" class="external-link">sbea</a></span><span class="op">(</span></span>
<span>    method <span class="op">=</span> <span class="st">'gsea'</span>, se <span class="op">=</span> <span class="va">edger</span>, gs <span class="op">=</span> <span class="va">sp_sigs</span>, alpha <span class="op">=</span> <span class="fl">0.1</span>, perm <span class="op">=</span> <span class="fl">0</span>,</span>
<span>    padj.method <span class="op">=</span> <span class="st">'BH'</span>, assay <span class="op">=</span> <span class="st">'relab_plus_1'</span></span>
<span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/EnrichmentBrowser/man/eaBrowse.html" class="external-link">gsRanking</a></span><span class="op">(</span><span class="va">edger_gsea</span>, signif.only <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="co">#&gt; DataFrame with 4 rows and 3 columns</span></span>
<span><span class="co">#&gt;                 GENE.SET      PVAL  ADJ.PVAL</span></span>
<span><span class="co">#&gt;              &lt;character&gt; &lt;numeric&gt; &lt;numeric&gt;</span></span>
<span><span class="co">#&gt; 1  aerophilicity:aerobic  0.000360  0.000762</span></span>
<span><span class="co">#&gt; 2 aerophilicity:facult..  0.000381  0.000762</span></span>
<span><span class="co">#&gt; 3 aerophilicity:anaero..  0.001980  0.002640</span></span>
<span><span class="co">#&gt; 4 aerophilicity:aeroto..  0.004690  0.004690</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="session-information">Session Information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.3.0 (2023-04-21)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 22.04.2 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; time zone: Etc/UTC</span></span>
<span><span class="co">#&gt; tzcode source: system (glibc)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] npGSEA_1.36.0                  GSEABase_1.62.0               </span></span>
<span><span class="co">#&gt;  [3] annotate_1.78.0                XML_3.99-0.14                 </span></span>
<span><span class="co">#&gt;  [5] AnnotationDbi_1.62.1           DESeq2_1.40.1                 </span></span>
<span><span class="co">#&gt;  [7] EnrichmentBrowser_2.30.1       graph_1.78.0                  </span></span>
<span><span class="co">#&gt;  [9] dplyr_1.1.2                    curatedMetagenomicData_3.8.0  </span></span>
<span><span class="co">#&gt; [11] TreeSummarizedExperiment_2.8.0 Biostrings_2.68.1             </span></span>
<span><span class="co">#&gt; [13] XVector_0.40.0                 SingleCellExperiment_1.22.0   </span></span>
<span><span class="co">#&gt; [15] SummarizedExperiment_1.30.1    Biobase_2.60.0                </span></span>
<span><span class="co">#&gt; [17] GenomicRanges_1.52.0           GenomeInfoDb_1.36.0           </span></span>
<span><span class="co">#&gt; [19] IRanges_2.34.0                 S4Vectors_0.38.1              </span></span>
<span><span class="co">#&gt; [21] BiocGenerics_0.46.0            MatrixGenerics_1.12.0         </span></span>
<span><span class="co">#&gt; [23] matrixStats_1.0.0              bugphyzz_0.0.1.3              </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;   [1] splines_4.3.0                 later_1.3.1                  </span></span>
<span><span class="co">#&gt;   [3] bitops_1.0-7                  filelock_1.0.2               </span></span>
<span><span class="co">#&gt;   [5] tibble_3.2.1                  taxizedb_0.3.1               </span></span>
<span><span class="co">#&gt;   [7] DirichletMultinomial_1.42.0   lifecycle_1.0.3              </span></span>
<span><span class="co">#&gt;   [9] edgeR_3.42.4                  bold_1.3.0                   </span></span>
<span><span class="co">#&gt;  [11] rprojroot_2.0.3               vroom_1.6.3                  </span></span>
<span><span class="co">#&gt;  [13] lattice_0.21-8                MASS_7.3-59                  </span></span>
<span><span class="co">#&gt;  [15] MultiAssayExperiment_1.26.0   magrittr_2.0.3               </span></span>
<span><span class="co">#&gt;  [17] limma_3.56.2                  sass_0.4.6                   </span></span>
<span><span class="co">#&gt;  [19] rmarkdown_2.22                jquerylib_0.1.4              </span></span>
<span><span class="co">#&gt;  [21] yaml_2.3.7                    httpuv_1.6.11                </span></span>
<span><span class="co">#&gt;  [23] DBI_1.1.3                     zlibbioc_1.46.0              </span></span>
<span><span class="co">#&gt;  [25] purrr_1.0.1                   RCurl_1.98-1.12              </span></span>
<span><span class="co">#&gt;  [27] yulab.utils_0.0.6             rappdirs_0.3.3               </span></span>
<span><span class="co">#&gt;  [29] GenomeInfoDbData_1.2.10       ggrepel_0.9.3                </span></span>
<span><span class="co">#&gt;  [31] irlba_2.3.5.1                 tidytree_0.4.2               </span></span>
<span><span class="co">#&gt;  [33] crul_1.4.0                    vegan_2.6-4                  </span></span>
<span><span class="co">#&gt;  [35] conditionz_0.1.0              pkgdown_2.0.7                </span></span>
<span><span class="co">#&gt;  [37] permute_0.9-7                 DelayedMatrixStats_1.22.0    </span></span>
<span><span class="co">#&gt;  [39] codetools_0.2-19              DelayedArray_0.26.3          </span></span>
<span><span class="co">#&gt;  [41] scuttle_1.10.1                xml2_1.3.4                   </span></span>
<span><span class="co">#&gt;  [43] tidyselect_1.2.0              httpcode_0.3.0               </span></span>
<span><span class="co">#&gt;  [45] ScaledMatrix_1.8.1            viridis_0.6.3                </span></span>
<span><span class="co">#&gt;  [47] BiocFileCache_2.8.0           jsonlite_1.8.4               </span></span>
<span><span class="co">#&gt;  [49] BiocNeighbors_1.18.0          decontam_1.20.0              </span></span>
<span><span class="co">#&gt;  [51] ellipsis_0.3.2                mia_1.8.0                    </span></span>
<span><span class="co">#&gt;  [53] scater_1.28.0                 systemfonts_1.0.4            </span></span>
<span><span class="co">#&gt;  [55] tools_4.3.0                   treeio_1.24.1                </span></span>
<span><span class="co">#&gt;  [57] ragg_1.2.5                    Rcpp_1.0.10                  </span></span>
<span><span class="co">#&gt;  [59] glue_1.6.2                    gridExtra_2.3                </span></span>
<span><span class="co">#&gt;  [61] xfun_0.39                     mgcv_1.8-42                  </span></span>
<span><span class="co">#&gt;  [63] withr_2.5.0                   BiocManager_1.30.20          </span></span>
<span><span class="co">#&gt;  [65] fastmap_1.1.1                 fansi_1.0.4                  </span></span>
<span><span class="co">#&gt;  [67] SparseM_1.81                  digest_0.6.31                </span></span>
<span><span class="co">#&gt;  [69] rsvd_1.0.5                    R6_2.5.1                     </span></span>
<span><span class="co">#&gt;  [71] mime_0.12                     textshaping_0.3.6            </span></span>
<span><span class="co">#&gt;  [73] colorspace_2.1-0              RSQLite_2.3.1                </span></span>
<span><span class="co">#&gt;  [75] utf8_1.2.3                    tidyr_1.3.0                  </span></span>
<span><span class="co">#&gt;  [77] generics_0.1.3                data.table_1.14.8            </span></span>
<span><span class="co">#&gt;  [79] DECIPHER_2.28.0               httr_1.4.6                   </span></span>
<span><span class="co">#&gt;  [81] S4Arrays_1.0.4                pkgconfig_2.0.3              </span></span>
<span><span class="co">#&gt;  [83] gtable_0.3.3                  blob_1.2.4                   </span></span>
<span><span class="co">#&gt;  [85] mgsub_1.7.3                   htmltools_0.5.5              </span></span>
<span><span class="co">#&gt;  [87] scales_1.2.1                  png_0.1-8                    </span></span>
<span><span class="co">#&gt;  [89] knitr_1.43                    tzdb_0.4.0                   </span></span>
<span><span class="co">#&gt;  [91] reshape2_1.4.4                uuid_1.1-0                   </span></span>
<span><span class="co">#&gt;  [93] nlme_3.1-162                  curl_5.0.0                   </span></span>
<span><span class="co">#&gt;  [95] cachem_1.0.8                  safe_3.40.0                  </span></span>
<span><span class="co">#&gt;  [97] stringr_1.5.0                 BiocVersion_3.17.1           </span></span>
<span><span class="co">#&gt;  [99] parallel_4.3.0                vipor_0.4.5                  </span></span>
<span><span class="co">#&gt; [101] desc_1.4.2                    pillar_1.9.0                 </span></span>
<span><span class="co">#&gt; [103] grid_4.3.0                    vctrs_0.6.2                  </span></span>
<span><span class="co">#&gt; [105] promises_1.2.0.1              BiocSingular_1.16.0          </span></span>
<span><span class="co">#&gt; [107] dbplyr_2.3.2                  beachmat_2.16.0              </span></span>
<span><span class="co">#&gt; [109] xtable_1.8-4                  cluster_2.1.4                </span></span>
<span><span class="co">#&gt; [111] beeswarm_0.4.0                Rgraphviz_2.44.0             </span></span>
<span><span class="co">#&gt; [113] evaluate_0.21                 KEGGgraph_1.60.0             </span></span>
<span><span class="co">#&gt; [115] locfit_1.5-9.7                cli_3.6.1                    </span></span>
<span><span class="co">#&gt; [117] taxize_1.0                    compiler_4.3.0               </span></span>
<span><span class="co">#&gt; [119] rlang_1.1.1                   crayon_1.5.2                 </span></span>
<span><span class="co">#&gt; [121] plyr_1.8.8                    fs_1.6.2                     </span></span>
<span><span class="co">#&gt; [123] ggbeeswarm_0.7.2              stringi_1.7.12               </span></span>
<span><span class="co">#&gt; [125] viridisLite_0.4.2             BiocParallel_1.34.2          </span></span>
<span><span class="co">#&gt; [127] munsell_0.5.0                 lazyeval_0.2.2               </span></span>
<span><span class="co">#&gt; [129] Matrix_1.5-4                  ExperimentHub_2.8.0          </span></span>
<span><span class="co">#&gt; [131] hoardr_0.5.3                  sparseMatrixStats_1.12.0     </span></span>
<span><span class="co">#&gt; [133] bit64_4.0.5                   taxa_0.4.2                   </span></span>
<span><span class="co">#&gt; [135] ggplot2_3.4.2                 statmod_1.5.0                </span></span>
<span><span class="co">#&gt; [137] KEGGREST_1.40.0               shiny_1.7.4                  </span></span>
<span><span class="co">#&gt; [139] interactiveDisplayBase_1.38.0 AnnotationHub_3.8.0          </span></span>
<span><span class="co">#&gt; [141] memoise_2.0.1                 bslib_0.4.2                  </span></span>
<span><span class="co">#&gt; [143] bit_4.0.5                     ape_5.7-1</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Samuel Gamboa.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
